dockers/docker-syncd-mlnx/start.sh

```diff
@@ -8,6 +8,10 @@ function clean_up {
 
 trap clean_up SIGTERM SIGKILL
 
+# fw-upgrade will exit if firmware was actually upgraded or if some error
+# occures
+. mlnx-fw-upgrade.sh
```

Comment:
> It is actually a trick to exit directly from mlnx-fw-upgrade.sh. Please add your comment as code comment.

There is a comment in start.sh line 11 and mlnx-fw-upgrade.sh line 51 regarding restart logic. Anything else needed?